rke2-rancher-kvm-tf/variable.tf

```diff
@@ -0,0 +1,56 @@
+####### RKE2 controlplane configuration ###########
+variable "rke2_server_ips" {
+  description = "list of rke2 server ip's"
+  type    = list(string)
+  default = ["192.168.100.151"]
+} 
+
+variable "rke2_server_memory" {
+  description = "Memory (in MiB) for rke2 controlplane"
+  default     = 4096
+}
+
+variable "rke2_server_vcpu" {
+  description = "Number of vCPUs for rke2 controlplane"
+  default     = 4
+}
+
+variable "rke2_server_disk_size" {
+  description = "Disk size (in Bytes) for rke2 controlplane"
+  default     = 1024 *1024 * 1024 * 25
+}
+
+variable "rke2_server_name" {
+  description = "The name of the rke2 masters"
+  default = "master"
+}
+
+######### Default Configuration for RKE2 cluster nodes ############
+
+variable "rke2_os_image_path" {
+  description = "Path to the operating system image"
+  #default  = "./ubuntu-16.04-server-cloudimg-amd64-disk1.img"
+  default = "http://cloud-images.ubuntu.com/releases/23.04/release-20230420/ubuntu-23.04-server-cloudimg-amd64.img"
```

Comment:
In `rke2-rancher-kvm-tf/terraform.tfvars` file, ubuntu 18.08 is used, all this cloud images version could point to one version variable